sale_exception/wizard/sale_exception_confirm_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="view_sale_exception_confirm" model="ir.ui.view">
+            <field name="name">Sale Exceptions</field>
+            <field name="model">sale.exception.confirm</field>
+            <field name="arch" type="xml">
+                <form string="Blocked in draft due to exceptions" version="7.0">
+                    <group>
+                        <field name="exception_ids" nolabel="1" colspan="4">
+                            <tree string="Sale Exceptions">
+                                <field name="name"/>
+                                <field name="description"/>
+                            </tree>
+                        </field>
+                        <newline/>
+                        <field name="ignore" 
+                            groups='sales_team.group_sale_manager'/>
+                    </group>
+                    <footer>
+                        <button name="action_confirm" string="_Close"
```

Comment:
string = `Close` and icon="gtk-ok" is working?

Reply:
create new button for close

Reply:
> create new button for close

yes but I know but `icon` and check your `string="_Close"` -> `string="Close"`